vendor/src/github.com/docker/distribution/registry/client/repository.go

```diff
@@ -345,9 +484,14 @@ func (ms *manifests) Delete(dgst digest.Digest) error {
 	if SuccessStatus(resp.StatusCode) {
 		return nil
 	}
-	return handleErrorResponse(resp)
+	return HandleErrorResponse(resp)
 }
 
+// todo(richardscothern): Restore interface and implementation with merge of #1050
```

Comment:
> > +// todo(richardscothern): Restore interface and implementation with merge of #1050
> This should be done or removed no?
> 

This is future work. The implementation was originally stubbed out, but I asked for it to be removed until an actual implementation exists.